app/controllers/podcast_episode_appearances_controller.rb

```diff
@@ -0,0 +1,114 @@
+class PodcastEpisodeAppearancesController < ApplicationController
+  before_action :authenticate_user!, only: %i[new create edit update destroy]
+  before_action :set_episode, only: %i[new]
+  before_action :set_podcast_episode_appearance, only: %i[show edit update destroy]
+  before_action :set_episode_with_strong_params, only: %i[create update]
+  before_action :set_podcast_episode_appearances, only: %i[index create update destroy]
```

Comment:
I don't think we need to call load appearances for create, update and destroy. 

I don't think we need the appearances of all episodes of all podcasts listed anywhere.

Trying to think about the final user experience helps in designing the functionality.